worker-images.yml

```diff
@@ -190,76 +194,68 @@ ronin_t_windows10_64_2009_prod:
     version: 1.0.0
     resource_group: rg-packer-worker-images
     deployment_id: "8346140"
-    SBOM: https://github.com/mozilla-platform-ops/worker-images/blob/main/config/win10-64-2009.md
     name: win10_64_2009
 ronin_t_windows10_64_2009_alpha:
   azure2:
     version: 1.0.0
     resource_group: rg-packer-worker-images
     deployment_id: alpha
-    SBOM: https://github.com/mozilla-platform-ops/worker-images/blob/main/config/win10-64-2009-alpha.md
     name: win10_64_2009_alpha
 # Windows 11
 ronin_b1_windows11_a64_24h2_builder_alpha:
   azure2:
     version: 0.0.1
     resource_group: rg-packer-worker-images
     deployment_id: alpha
-    SBOM: https://github.com/mozilla-platform-ops/worker-images/blob/main/config/win11-a64-24h2-builder-alpha.md
     name: win11_a64_24h2_builder_alpha
 ronin_b1_windows11_a64_24h2_builder:
   azure2:
     version: 1.0.0
     resource_group: rg-packer-worker-images
     deployment_id: a03478d
-    SBOM: https://github.com/mozilla-platform-ops/worker-images/blob/main/config/win11-a64-24h2-builder.md
     name: win11_a64_24h2_builder
+    sbom: false
```

Comment:
I see that this, and at least one other entry, appear to be losing SBOMs? That seems less than ideal? (Maybe there's a good reason though?)

Reply:
The old `SBOM` entries were added manually and were unused, so they were essentially just a comment. Now the sboms are moving to a new naming scheme. For whatever reason the url for this one was a 404, so I disabled it for now. I suspect jmoss just needs to tweak something in worker-images, but this way the PR can land and he can re-enable it when it's ready.